### DISCUSSION TITLE
Any reasons for empty module headers with `?mid=<ModuleID>` ?

### DISCUSSION THREAD

**Discussion by roman-yagodin** [dnnsoftware/Dnn.Platform#4049] [General]
I've noticed that DNN is showing empty module headers for default view controls (regardless of module or site container settings) then passing `?mid=<ModuleID>` argument via querystring.

I can imagine some use cases for links to the separate modules on the page, but considering the way it works now
users could miss some important information about the module. Is there any design reasons for showing empty module header, or this is kind of default behavior which could be altered safely?

Some thoughts:

- Default titles for other types of controls could be set via `ControlTitle_<ControlKey>.Text` resource strings, but default view controls must (?) have empty control keys in manifest, so providing just `ControlTitle.Text` or `ControlTitle_.Text` didn't work (at least for me).
- The module itself can detect the situation and alter its container control header.
- Custom theme can detect the situation and provide alternative way to show module title.

**Comment by mitchelsellers** [comment]
?mid= should only really be passed in situation where you are trying to isolate a single module control.  As that starts the triggering of isolation mode.
